Nodejs/Product/TargetsVsix/Microsoft.NodejsToolsV2.targets

```diff
@@ -0,0 +1,312 @@
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Specifies the directories which are ignored.  Any paths which contain these directory names
+         will be ignored.  The directory name must completely match and no wild cards are allowed
+     -->
+    <AnalysisIgnoredDirectories Condition="'$(AnalysisIgnoredDirectories)' == ''">bower_components;node_modules\core-js</AnalysisIgnoredDirectories>
+    <DockerExtraPublishProfileProperties>ExcludeFoldersFromDeployment=node_modules</DockerExtraPublishProfileProperties>
+  </PropertyGroup>
+  <!-- *********************************************************************************** -->
+  <!-- Normal install environment, use the current version -->
+  <PropertyGroup>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <NodejsSupportPath>$(MSBuildThisFileDirectory)</NodejsSupportPath>
+    <ProjectHome Condition="'$(ProjectHome)' == ''">.</ProjectHome>
+    <DebugSymbols>false</DebugSymbols>
+    <FakesImported>true</FakesImported>
+    <!-- disables quality tools fakes imports -->
+  </PropertyGroup>
+
+  <!-- Gets the fully qualified path to the project home.  -->
+  <PropertyGroup>
+    <QualifiedProjectHome>$(MSBuildProjectDirectory)</QualifiedProjectHome>
+    <QualifiedProjectHome Condition="'$(ProjectHome)' != '' and '$(ProjectHome)' != '.'">$([System.IO.Path]::GetFullPath($([System.IO.Path]::Combine($(MSBuildProjectDirectory), $(ProjectHome)))))</QualifiedProjectHome>
+    <QualifiedProjectHome Condition="!HasTrailingSlash('$(QualifiedProjectHome)')">$(QualifiedProjectHome)\</QualifiedProjectHome>
+  </PropertyGroup>
+
+  <Target Name="GetTargetFrameworkVersion" Returns="$(TargetFrameworkVersion)" />
+  <Target Name="GetTargetFrameworkDirectories" Returns="" />
+  <Target Name="GetTargetFrameworkMoniker" Returns="" />
+  <Target Name="GetTargetFrameworkMonikerDisplayName" Returns="" />
+  <Target Name="CreateManifestResourceNames" Returns=""/>
+
+  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets"/>
+  <Import Project="$(VSToolsPath)\Docker Tools\Microsoft.DockerTools.Publish.targets" Condition="Exists('$(VSToolsPath)\Docker Tools\Microsoft.DockerTools.Publish.targets')"/>
+
+  <Target
+    Name="GetReferenceAssemblyPaths"
+    DependsOnTargets="$(GetReferenceAssemblyPathsDependsOn)">
+    <!-- We have no reference assembly paths -->
+  </Target>
+
+  <ItemGroup>
+    <NodeModulesContent Include="$(ProjectHome)\node_modules\**\*.*" />
+    <TypeScriptSource Exclude="$(ProjectHome)\node_modules\**\*.*;$(ProjectHome)\obj\**\*.*;$(ProjectHome)\bin\**\*.*;$(ProjectHome)\**\*.d.ts" Include="$(ProjectHome)\**\*.ts;$(ProjectHome)\**\*.tsx;$(ProjectHome)\**\*.jsx;" />
+  </ItemGroup>
+
+  <Target Name="CopyFilesToOutputDirectory"/>
```

Comment:
><Target Name="CopyFilesToOutputDirectory"/> [](start = 2, length = 43)

I thought we still wanted to copy JS files to the output directory.  Does that happen through a different mechanism?  Or do we consider having a separate output directory redundant?

Reply:
If the target is not overridden VS tries to copy the ProjectnName.exe from obj. This scenario is not useful for Node projects so it got completely removed.

The output is configured by `tsconfig.json`.